src/js/tracks/html-track-element-list.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file html-track-element.js
+ */
+
+import * as browser from '../utils/browser.js';
+import document from 'global/document';
+
+class HtmlTrackElementList {
+  constructor(trackElements = []) {
+    let list = this;
+
+    if (browser.IS_IE8) {
+      list = document.createElement('custom');
+
+      for (let prop in HtmlTrackElementList.prototype) {
+        list[prop] = HtmlTrackElementList.prototype[prop];
```

Comment:
Looks like not all got updated by this PR: https://github.com/videojs/video.js/pull/2565/files

Not sure what you mean by the second part but babel will always compile it down to prototype stuff. Though, the prototype still exists in full classes as well, I believe.